src/globals.c

```diff
@@ -158,9 +165,13 @@ static void zpc_init(void)
 	assert(rc == 0);
 	if (load_ep11_library(&ep11, true) != 0)
 		DEBUG("loading EP11 library failed");
-	else
+	else {
```

Comment:
Style: If the else has { ... } then the if should also have it.

Reply:
Done.